crates/kitsune_p2p/types/src/config.rs

```diff
@@ -100,11 +100,14 @@ pub mod tuning_params_struct {
         /// Default timeout for rpc single. [Default: 30s]
         default_rpc_single_timeout_ms: u32 = 1000 * 30,
 
-        /// Default agent count for rpc multi. [Default: 2]
-        default_rpc_multi_remote_agent_count: u32 = 2,
-
-        /// Default timeout for rpc multi. [Default: 30s]
-        default_rpc_multi_timeout_ms: u32 = 1000 * 30,
+        /// Default agent count for rpc multi. [Default: 3]
+        default_rpc_multi_remote_agent_count: u8 = 3,
```

Comment:
Is this what is overwritten to 1? Maybe that's a little misleading?

Reply:
It's only overwritten for `get_links` and `get_agent_activity`..

`get` and `get_meta` still use the default of 3.